stdlib/bir/src/main/ballerina/src/bir/bir_model.bal

```diff
@@ -411,6 +411,7 @@ public type BUnionType record {|
 
 public type BTupleType record {|
    BType?[]  tupleTypes;
+   BType?  restType;
```

Comment:
```suggestion
   BType?  restType = ();
```

We can set `()` as the initial value, keep https://github.com/ballerina-platform/ballerina-lang/pull/17008/files#diff-c55ac83ca455bbaf05c3333c10c991bdL210 as is and set https://github.com/ballerina-platform/ballerina-lang/pull/17008/files#diff-c55ac83ca455bbaf05c3333c10c991bdR214 only if `restPresent` is true.

Reply:
Fixed in 22233bb